Core/Searcher/SeedSearcher.cpp

```diff
@@ -26,6 +26,8 @@ SeedSearcher::SeedSearcher(const QVector<Pokemon> &pokemon, const QVector<int> &
     firstResult(firstResult),
     searching(false)
```

Comment:
Can you move it the initializer list right below searching?

Reply:
done